### PR TITLE
Use updated (working) diffblue.yml file

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: diffblue.yml [hunk 1]
```diff
@@ -1,4 +1,2 @@
-branch: master
-language: java
-buildCmd: "mvn build"
-testCmd: "mvn cobertura:cobertura"
+buildCmd: "mvn test"
```
Comment (rjmunro) [inline diffblue.yml:1]: This should be `mvn compile`.

`mvn test` will mean that the existing tests are built and then analysed, so we will generate tests to test tests, which is a bit of a mess.

Comment (ghost) [reply diffblue.yml:1]: mvn compile doesn't run. It's complaining of a missing dependency which I can't seem to resolve myself.

Comment (rjmunro) [reply diffblue.yml:1]: If it's complaining about a missing dependency, `mvn test` must only be compiling a subset of the code, which means we will only generate tests for that subset, which, ironically, will be the part of the code that already has tests.

We need to fix the dependency issue.

Comment (ghost) [reply diffblue.yml:1]: This is the error i'm getting when I run mvn compile: 
`Failed to execute goal on project cassandra-driver-mapping: Could not resolve dependencies for project com.datastax.cassandra:cassandra-driver-mapping:bundle:3.2.0-SNAPSHOT: Failure to find com.datastax.cassandra:cassandra-driver-core:jar:tests:3.2.0-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots was cached in the local repository, resolution will not be reattempted until the update interval of sonatype-nexus-snapshots has elapsed or updates are forced`.

I should point out it's happening on datastax's own 3.x branch too.

Comment (rjmunro) [reply diffblue.yml:1]: Are there any datastax branches where it doesn't happen? If so, it's pretty easy to find the problem using `git bisect`. Give me a shout and I'll show you how.

Comment (ghost) [reply diffblue.yml:1]: Yes, i've got one thanks. I've updated master to point to the most-recent working branch (from tag 3.1.4). mvn compile now works successfully

**Pull request merged**